src/lib/permissions/clients/index.ts

```diff
@@ -10,6 +10,16 @@ export * from '../proposals/client/proposalPermissionsHttpClient';
 export * from '../proposals/availableProposalCategoryPermissions.class';
 export * from '../proposals/availableProposalPermissions.class';
 
+// Pages clients
+export * from '../pages/client/interfaces';
```

Comment:
these should be namespaced as well. If you look, this file gets exported by /shaerd which gets exported by /index.ts, so this is actually exporting everything at the top of @charmverse/core.